src/features/teams/team.service.ts

```
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { TeamEntity } from './entities/team.entity';
+
+
+@Injectable()
+export class TeamService {
+    
+    constructor(
+        @InjectRepository(TeamEntity)
+        private _teamRepository: Repository<TeamEntity>,
+    ){}
+
+    async findAll(): Promise<TeamEntity[]> {
+        return this._teamRepository.find();
+    }
+
+    async findOne(id: number): Promise<TeamEntity | undefined> {
```

throw an error instead of returning undefied in case the entity was not found